modules/installation/pages/install-vm.adoc

```diff
@@ -65,28 +69,24 @@ These disks will be partitioned and mounted in <<proc.jeos.susemgr.prep>>.
 | VirtIO Disk 4        | swap      | 4{nbsp}GB
 |===
 
-. Click menu:Begin Installation[] and your new VM will boot from the JeOS image.
+. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
 
 Proceed through the basic JeOS installation prompts until you reach the command line.
 
-.Root Password
-[TIP]
-====
 During the basic installation prompts you are asked to enter the root password.
-Select a strong password and then in the next message box btn:[Confirm root Password].
-====
+Select a strong password and in the next message box click btn:[Confirm root Password].
```

Comment:
"Select" probably isn't the right verb here. Possibly "type"?

Reply:
I think we can shorten it.  Maybe, somewhere else we should talk about passwords in general.